HyperV/scripts/create-environment.ps1

```diff
@@ -306,6 +363,50 @@ Copy-Item -Recurse $configDir "$remoteConfigs\$hostname"
 
 Write-Host "Starting the services"
 
+
+
+if ($jobType -eq 'smbfs')
+{
+    $currDate = (Get-Date).ToString()

```

Comment:
Just a small suggestion, I think that all this workflow could be wrapped in a function, but we can do that later.